### ISSUE TITLE
MarkdownView doesn't work

### ISSUE THREAD

**Issue by LittleHans8** [mukeshsolanki/MarkdownView-Android#7]
When I run in my network callback,it seems doesn't work.The code like this:

 ```
  @Override public void respondSuccess(final Trees data) { // my netwrokCallback
          Thread thread = new Thread(new Runnable() {
      @Override public void run() {
        try {
          // get markdown txt from bootstrap
          Blob blob = mGitDateService.getBlob("twbs", "bootstrap",
              "e0e7c32bdd7d6d91efc36ad9da1a39549c9b0e13").execute().body();
          final String txtMarkdown = new String(Base64.decode(blob.content, Base64.DEFAULT));
          Log.d(TAG, "run: " + txtMarkdown); // log->success
          getActivity().runOnUiThread(new Runnable() {
            @Override public void run() {
              mTxtDescription.setText(txtMarkdown); // I test the code in a TextView, show normal
              mMarkdownView.setMarkDownText(txtMarkdown); // show abnormal,the markdown is blank
              Log.d(TAG, "run: " + "already set markdownTxt"); // log->success
            }
          });
        } catch (IOException e) {
          e.printStackTrace();
        }
      }
    });
    thread.start();
  }
 ```

**Comment by LittleHans8** [comment]
@tajchert was already solved this issue!  see this pull requests #5

**Comment by mukeshsolanki** [comment]
@LittleHans8 Fixed and released in version 1.0.2 @tajchert thanks
